docs/uma-tokenholders/guidence-on-collateral-currency-addition.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Guidance for the Addition of Collateral Currencies
+sidebar_label: Guidance for the Addition of Collateral Currencies
+---
+Collateral currencies can be added through the collateral currency UMIP template available on Github, by adding 
+
+ - the name of the token to the be added 
+ - the contract address of the token  together with a link to the address on etherscan
+ - the amount that should be added to the store as final fee expressed in units of the intended collateral currency.   
+_This should equate to approximately $400 at the time the UMIP is submitted.  If the token is not directly listed on Coingecko and/or Coinmarketcap, please add how the estimated dollar value of the token was obtained in the Rationale section._
+
+***Note**: rebasing tokens cannot be added as collateral currencies as they do not work with UMA's factory contracts*
+
+____________
+The below provides some guidelines when using collateral currencies in an UMA contract
+
+### Stablecoins
+
+Consider carefully whether to treat stablecoins as their intended peg, or as the current market value.
```

Comment:
Can we provide some more information here? Maybe something like:

```
UMA collateral types within most registered financial contracts are treated pricelessly. That is to say that a financial contract's price identifier does not necessarily take into account the USD denominated value of a stablecoin or other collateral. This means that a price in an UMA financial contract only determines the units of collateral that a synthetic token is worth.

Because of this, if it is wished to use a stablecoin as collateral and account for its fluctuations away from a $1 peg, the price identifier used should also account for the stablecoin's price in USD (or other fiat currency). As an example, an Expiring Multiparty contract that creates synthetic ETH with USDT as collateral will need to define a price identifier similar to the [ETHUSD price identifier](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-6.md). With the ETHUSD price identifier defined as is and ETHUSD at 2500, each synthetic token will be worth 2500 USDT. If a contract deployer wishes to reflect the true ETHUSDT price, they will need to define a new price identifier that determines both the ETH/USD and the USD/USDT prices and multiplies the two.

This information does not necessarily need to be contained in a collateral currency UMIP.
```

Reply:
Added in.